README.md

```diff
@@ -176,6 +176,13 @@ Build steps:
    - cmake ..
    - sudo make -j4 install
 
+3. Build cppzmq via [vcpkg](https://github.com/Microsoft/vcpkg/). This does an out of source build and installs the build files
+   - git clone https://github.com/Microsoft/vcpkg.git
+   - cd vcpkg
+   - ./bootstrap-vcpkg.sh
+   - ./vcpkg integrate install
+   - ./vcpkg install cmark
```

Comment:
Is this correct?

Reply:
Yes, it's fully usable. It's just that the script name varies a little between systems. 
The script name :
````
 `linux` OR `osx`: bootstrap-vcpkg.sh
 `Windows`: bootstrap-vcpkg.bat
 ```` 
 Maybe I should add a note to this, what do you think? 🧐
 Below is my full log running on `x64-linux`: (I can provide logs from other systems and any information about the build if needed)
 ````
 test@test002:~$ ls
bug  other  test
test@test002:~$ cd test/
test@test002:~/test$ ls
test@test002:~/test$ git clone https://github.com/Microsoft/vcpkg.git
Cloning into 'vcpkg'...
remote: Enumerating objects: 163385, done.
remote: Counting objects: 100% (124/124), done.
remote: Compressing objects: 100% (102/102), done.
remote: Total 163385 (delta 55), reused 50 (delta 22), pack-reused 163261
Receiving objects: 100% (163385/163385), 61.38 MiB | 26.73 MiB/s, done.
Resolving deltas: 100% (104806/104806), done.
test@test002:~/test$ cd vcpkg
test@test002:~/test/vcpkg$ ./bootstrap-vcpkg.sh
Downloading vcpkg-glibc...
Telemetry
---------
vcpkg collects usage data in order to help us improve your experience.
The data collected by Microsoft is anonymous.
You can opt-out of telemetry by re-running the bootstrap-vcpkg script with -disa bleMetrics,
passing --disable-metrics to vcpkg on the command line,
or by setting the VCPKG_DISABLE_METRICS environment variable.

Read more about vcpkg telemetry at docs/about/privacy.md
test@test002:~/test/vcpkg$ ./vcpkg integrate install
Applied user-wide integration for this vcpkg root.
CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=/home/test/test/vcpkg/scripts/buildsystems/vcpkg.cmake"
test@test002:~/test/vcpkg$ ./vcpkg install cmark
Computing installation plan...
A suitable version of cmake was not found (required v3.24.0). Downloading portable cmake v3.24.0...
Downloading cmake...
  https://github.com/Kitware/CMake/releases/download/v3.24.0/cmake-3.24.0-linux-x86_64.tar.gz -> /home/test/test/vcpkg/downloads/cmake-3.24.0-linux-x86_64.                                          tar.gz
Extracting cmake...
The following packages will be built and installed:
    cmark[core]:x64-linux -> 0.30.1
  * vcpkg-cmake[core]:x64-linux -> 2022-07-18
  * vcpkg-cmake-config[core]:x64-linux -> 2022-02-06#1
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
Restored 2 package(s) from /home/test/.cache/vcpkg/archives in 315.4 ms. Use --debug to see more details.
Installing 1/3 vcpkg-cmake:x64-linux...
Elapsed time to handle vcpkg-cmake:x64-linux: 791.7 us
Installing 2/3 vcpkg-cmake-config:x64-linux...
Elapsed time to handle vcpkg-cmake-config:x64-linux: 568.2 us
Installing 3/3 cmark:x64-linux...
Building cmark[core]:x64-linux...
-- Downloading https://github.com/commonmark/cmark/archive/977b128291c0cf6c5053cdcf2ac72e627f09c105.tar.gz -> commonmark-cmark-977b128291c0cf6c5053cdcf2ac72e627f09c105.tar.gz...
-- Extracting source /home/test/test/vcpkg/downloads/commonmark-cmark-977b128291c0cf6c5053cdcf2ac72e627f09c105.tar.gz
-- Applying patch add-feature-tools.patch
-- Using source at /home/test/test/vcpkg/buildtrees/cmark/src/627f09c105-7046f2860b.clean
-- Configuring x64-linux
-- Building x64-linux-dbg
-- Building x64-linux-rel
-- Fixing pkgconfig file: /home/test/test/vcpkg/packages/cmark_x64-linux/lib/pkgconfig/libcmark.pc
-- Fixing pkgconfig file: /home/test/test/vcpkg/packages/cmark_x64-linux/debug/lib/pkgconfig/libcmark.pc
-- Installing: /home/test/test/vcpkg/packages/cmark_x64-linux/share/cmark/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: "/home/test/.cache/vcpkg/archives/f6/f6f2998d20e932fce49f86f9425a2526d868aa7e4fa2047ea1d43744d5e56701.zip"
Elapsed time to handle cmark:x64-linux: 6.39 s

Total elapsed time: 20.94 s

cmark provides CMake targets:

    # this is heuristically generated, and may not be correct
    find_package(cmark CONFIG REQUIRED)
    target_link_libraries(main PRIVATE cmark::cmark_static)

test@test002:~/test/vcpkg$
````

Reply:
Ok, but I am a but confused why install cmark (commonmark-cmark), shouldn't that be cppzmq?

Reply:
Sorry, I had a "brain fade"when I left this comment. 😣

````
test@test002:~/test/vcpkg$ ./vcpkg install cppzmq
Computing installation plan...
The following packages will be built and installed:
    cppzmq[core]:x64-linux -> 4.8.1
  * zeromq[core]:x64-linux -> 4.3.4#6
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
Restored 0 package(s) from /home/test/.cache/vcpkg/archives in 144.5 ms. Use --debug to see more details.
Installing 1/2 zeromq:x64-linux...
Building zeromq[core]:x64-linux...
-- Downloading https://github.com/zeromq/libzmq/archive/v4.3.4.tar.gz -> zeromq-libzmq-v4.3.4.tar.gz...
-- Extracting source /home/test/test/vcpkg/downloads/zeromq-libzmq-v4.3.4.tar.gz
-- Applying patch fix-arm.patch
-- Applying patch zeromq-libzmq-4310-64e6d37ab8.diff
-- Applying patch zeromq-libzmq-4311-2b04e0ce47.diff
-- Using source at /home/test/test/vcpkg/buildtrees/zeromq/src/v4.3.4-933c603e1e.clean
-- Configuring x64-linux
CMake Warning at installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:302 (message):
  The following variables are not used in CMakeLists.txt:

      WITH_PERF_TOOL

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/zeromq/portfile.cmake:28 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building x64-linux-dbg
-- Building x64-linux-rel
-- Fixing pkgconfig file: /home/test/test/vcpkg/packages/zeromq_x64-linux/lib/pkgconfig/libzmq.pc
-- Fixing pkgconfig file: /home/test/test/vcpkg/packages/zeromq_x64-linux/debug/lib/pkgconfig/libzmq.pc
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: "/home/test/.cache/vcpkg/archives/f2/f25e487cc0a9d0cec27a491128e50e63010cce81a2ad755662ee43fcc098264c.zip"
Elapsed time to handle zeromq:x64-linux: 29.96 s
Installing 2/2 cppzmq:x64-linux...
Building cppzmq[core]:x64-linux...
-- Downloading https://github.com/zeromq/cppzmq/archive/v4.8.1.tar.gz -> zeromq-cppzmq-v4.8.1.tar.gz...
-- Extracting source /home/test/test/vcpkg/downloads/zeromq-cppzmq-v4.8.1.tar.gz
-- Using source at /home/test/test/vcpkg/buildtrees/cppzmq/src/v4.8.1-6edef9e3e5.clean
-- Configuring x64-linux
-- Building x64-linux-dbg
-- Building x64-linux-rel
-- Installing: /home/test/test/vcpkg/packages/cppzmq_x64-linux/share/cppzmq/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: "/home/test/.cache/vcpkg/archives/d7/d766fb2b4313c4a451cf11cd666c99895d0419e103aab0e9edb55e0f6e34f77f.zip"
Elapsed time to handle cppzmq:x64-linux: 1.468 s

Total elapsed time: 31.96 s

cppzmq provides CMake targets:

    # this is heuristically generated, and may not be correct
    find_package(cppzmq CONFIG REQUIRED)
    target_link_libraries(main PRIVATE cppzmq cppzmq-static)

test@test002:~/test/vcpkg$
````

Reply:
No problem, just need to fix the PR also.